backend/app/api/endpoints/admin.py

```
@@ -1707,3 +1715,643 @@ async def delete_public_retriever(
+    ghost_ref = spec.get("ghostRef", {})
+    ghost_name = ghost_ref.get("name")
+    ghost_namespace = ghost_ref.get("namespace", "default")
+    if ghost_name:
+        ghost = (
+            db.query(Kind)
+            .filter(
+                Kind.user_id == 0,
+                Kind.kind == "Ghost",
+                Kind.name == ghost_name,
+                Kind.namespace == ghost_namespace,
+                Kind.is_active == True,
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)